pkg/signals/signal.go

```diff
@@ -0,0 +1,79 @@
+/*
+Copyright 2021 The Kubernetes Authors.
```

Comment:
Why Kuberentes Authors? If dependents on k8s, let's import it.

Reply:
fixed. import it from apiserver